index.html

```diff
@@ -29,18 +29,34 @@
 
   <form class="uk-search uk-search-default margin-medium">
     <div class="uk-margin">
-      <label for="searching-field" class="font-size-medium">WHAT ARE YOU LOOKING FOR?</label>
+      <label for="searching-field" class="font-size-small">WHAT ARE YOU LOOKING FOR?</label>
       <input id="searching-field" class="uk-search-input" type="search" placeholder="Search...">
       <button type="submit" class="uk-search-icon-flip" uk-search-icon></button>
-
     </div>
   </form>
 
-  <!-- zrob ula i wyswietl najprostsze lisy z tytulami -->
-
-  <ul id="recipe-list">
+  <div class="uk-grid uk-child-width-expand">
+      <div>
+        <ul id="recipe-list"></ul>
+      </div>
 
-  </ul>
+      <div id="display-recipe" class="blue-border recipe-box">
+        <h1 class="font-size-huge">Spaghetti - title</h1>
+        <h2 class="font-size-large">Ingredients - automatic</h2>
+        <ul>
+          <li>składnik</li>
+          <li>składnik</li>
+        </ul>
+        <h2 class="font-size-large">Instructions - automatic</h2>
+        <h3 class="font-size-medium">Instructions goes here</h3>
```

Comment:
Instructions - a number of paragraphs written in prose I presume - inside a heading? That doesn't sound right, does it?